packages/report-server/src/reportBuilder/transform/functions/insertRows.ts

```diff
@@ -43,8 +42,7 @@ const insertRows = (rows: Row[], params: InsertParams): Row[] => {
     }
     const newRow: Row = {};
     Object.entries(params.columns).forEach(([key, expression]) => {
-      const [newRowKey, newRowValue] = getParsedColumnKeyAndValue(key, expression, parser);
-      newRow[newRowKey] = newRowValue;
+      newRow[parser.evaluate(key)] = parser.evaluate(expression);
```

Comment:
Love that it not only simplifies the rules for users, it also simplifies our code